SpeechSandbox/Assets/Scripts/SpeechSandboxStreaming.cs

```diff
@@ -34,52 +34,140 @@ public class SpeechSandboxStreaming : MonoBehaviour
     public AudioClip sorryClip;
     public List<AudioClip> helpClips;
 
-
-
     [SerializeField]
     private fsSerializer _serializer = new fsSerializer();
-    private SpeechToText _speechToText;
-    private Conversation _conversation;
 
+    #region PLEASE SET THESE VARIABLES IN THE INSPECTOR
+    [Header("Speech To Text")]
+    [Tooltip("The service URL (optional). This defaults to \"https://stream.watsonplatform.net/speech-to-text/api\"")]
+    [SerializeField]
+    private string stt_serviceUrl = "";
+    [Header("CF Authentication")]
+    [Tooltip("The authentication username.")]
+    [SerializeField]
     private string stt_username = "";
-    private string stt_password = "";
-    // Change stt_url if different from below
-    private string stt_url = "https://stream.watsonplatform.net/speech-to-text/api";
+    [Tooltip("The authentication password.")]
+    [SerializeField]
+    private string stt_password;
+    [Header("IAM Authentication")]
+    [Tooltip("The IAM apikey.")]
+    [SerializeField]
+    private string stt_iamApikey;
+    [Tooltip("The IAM url used to authenticate the apikey (optional). This defaults to \"https://iam.bluemix.net/identity/token\".")]
+    [SerializeField]
+    private string stt_iamUrl;
+
+    [Header("Watson Assistant")]
+    [Tooltip("The service URL (optional). This defaults to \"https://gateway.watsonplatform.net/conversation/api\"")]
```

Comment:
Should prob change this URL to Assistant

Reply:
done.